src/javascript/app/pages/trade/markets.jsx

```diff
@@ -399,30 +444,87 @@ class Markets extends React.Component {
                     <div className='markets_view'>
                         <div className='markets_column'>
                             <div className='desktop'>
-                                {markets.map(([key, obj]) =>
-                                    <div
-                                        className={`market ${active_market === key ? 'active' : ''}`}
-                                        key={key}
-                                        onClick={scrollToMarket.bind(null,`${key}`)}
-                                    >
-                                        <span className={`icon ${key} ${active_market === key ? 'active' : ''}`} />
-                                        <span>{obj.name}</span>
+                                {Object.keys(sorted_markets).map((item) => (
+                                    <div key={item}>
+                                        {item === 'None' ? (
```

Comment:
This here also uses `None` which is display_name that gets translated. Can we pass `subgroup` instead or together and use it for comparison?

Reply:
Resolved